test/image-blueprints-bootc/layer2-source/group1/microshift-sshd-ownership.conf.template

```diff
@@ -0,0 +1,4 @@
+# Change group of ssh key files if they are not owned by root already because
+# that is the new expected default upstream
+[Service]
+ExecStartPre=/usr/bin/find /etc/ssh/ -name "ssh_*_key" -a '!' -gid 0 -exec /bin/chgrp ssh_keys {} +
```

Comment:
Test agent already does it, right?

Reply:
Ah, that's a good point!
I will remove the SSH settings and fix the test agent not to do update according to the new guidelines.

Reply:
Done